build/ci/github_actions/operator/deploy_driver.sh

```diff
@@ -25,9 +25,8 @@ edit_cr_images (){
 
 edit_operator_yaml_image (){
   cd $(dirname $operator_yaml)
-  operator_image_in_branch=`yq eval '(.spec.template.spec.containers[0].image | select(. == "*ibmcom*"))' $(basename $operator_yaml)`
-  # CSI-3223
-  sed -i "s+$operator_image_in_branch+$operator_image_for_test+g" $(basename $operator_yaml)
+  operator_image_in_branch=`yq eval '(. | select(.kind == "Deployment") | .spec.template.spec.containers[0].image)' $(basename $operator_yaml)`
+  sed -i "s+$operator_image_in_branch+$operator_image_for_test+g" $(basename $operator_yaml) ## TODO: CSI-3223 need to edit the operator image only in a specifics places
```

Comment:
I suggest writing the information in the ticket since it's easier to maintain.

Reply:
the information is in the ticket

Reply:
Then we can remove it from the code.
We want one place for managing the ticket information.

Reply:
@oriyarde [asked](https://github.com/IBM/ibm-block-csi-operator/pull/189#discussion_r686721282) for it.
@oriyarde this is what you meant? that the information will be there?

Reply:
yes, because there [wasn't even a `TODO: `](https://github.com/IBM/ibm-block-csi-operator/blob/4d9db615055efce03c87321e7f539e2ed285d22a/build/ci/github_actions/operator/deploy_driver.sh#L29)

Reply:
@oriyarde I don't think the `TODO:` is the problem that @zingero mentioned, it is the information after the `TODO:`.
So do we need also the information or can I remove it?

Reply:
@zingero could we keep at least three~ words, so we know the general intention here? e.g. `avoid using sed`

Reply:
Yes, we can

Reply:
thanks

Reply:
> Yes, we can

president Zinger

Reply:
done

Reply:
@zingero please resolve all conversations which you opened before approving (unless they are intentionally left open for a future ticket or something)